src/modules/edc-demo/pages/catalog-browser/asset-details/asset-details.component.html

```diff
@@ -70,27 +70,33 @@ <h2>Policy Details</h2>
   </div>
 
   <mat-list dense class="catalogue-meta-data-list" fxLayout="row wrap">
-    <mat-list-item>
-      <mat-icon mat-list-icon>policy</mat-icon>
-      <div mat-line class="asset-property-name">Policy ID</div>
-      <div mat-line>{{contractOffer.id}}</div>
-    </mat-list-item>
-    <mat-list-item>
-      <mat-icon mat-list-icon>ads_click</mat-icon>
-      <div class="asset-property-name" mat-line>Target</div>
-      <div mat-line>{{contractOffer.policy.target ?? '- -'}}</div>
-    </mat-list-item>
     <mat-list-item>
       <mat-icon mat-list-icon>handshake</mat-icon>
-      <div class="asset-property-name" mat-line>Verpflichtungen</div>
-      <div mat-line *ngFor="let obligation of contractOffer.policy.obligations!">{{obligation.action?.type}} {{obligation.target}}</div>
+      <div class="asset-property-name" mat-line>Obligations</div>
+      <div mat-line *ngFor="let obligation of contractOffer.policy.obligations!">{{ obligation.action?.type }} {{ obligation.target }}</div>
       <div mat-line *ngIf="contractOffer.policy.obligations?.length === 0">- -</div>
     </mat-list-item>
+    <mat-list-item>
+      <mat-icon mat-list-icon>lock</mat-icon>
+      <div class="asset-property-name" mat-line>Prohibitions</div>
+      <div mat-line *ngFor="let prohibition of contractOffer.policy.prohibitions!"></div>
+      <div mat-line *ngIf="contractOffer.policy.prohibitions?.length === 0">- -</div>
+    </mat-list-item>      
     <mat-list-item>
       <mat-icon mat-list-icon>approval</mat-icon>
       <div class="asset-property-name" mat-line>Permissions</div>
-      <div mat-line *ngFor="let permission of contractOffer.policy!.permissions!">{{permission?.action?.type}} {{permission.target}}</div>
-    </mat-list-item>
+      <div mat-line *ngFor="let permission of contractOffer.policy!.permissions!">
+        USE {{ permission.target }} with
```

Comment:
"USE" ist eine Action, die ja verschieden sein kann. Der Wert steht in `permission.action.type`.

Reply:
https://github.com/agri-gaia/DataDashboard/pull/10/commits/ac756d49d3e2f8e679a6a4cfd02390caea83b3f2